app.rb

```diff
@@ -0,0 +1,169 @@
+['./book_class', './classroom_class', './person_class', './rental_class', './student_class',
+ './teacher_class'].each do |file|
+  require file
+end
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @classroom = Classroom.new('1-A')
+    @rentals = []
+    @people = []
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def options(option)
+    case option
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rental
+    when 7
+      puts 'Thanks for using the app :)'
+    else
+      puts 'Invalid option.'
+      menu
+    end
+  end
+
+  # rubocop:enable Metrics/CyclomaticComplexity
+  def menu
+    puts "\n\nPlease choose an option by enterin a number:"
+    puts '1.- List all books'
+    puts '2.- List all people'
+    puts '3.- Create a person'
+    puts '4.- Create a book'
+    puts '5.- Create a rental'
+    puts '6.- List all rentals for a given person id'
+    puts '7.- Exit'
+    print '> '
+    option = gets.chomp.to_i
+    options(option)
+  end
+
+  def list_books
+    if @books.empty?
+      puts "\nThere are not available books"
+      menu
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts "\nThere are not people to show"
+      menu
+    else
+      @people.each do |person|
+        type = if person.is_a?(Teacher)
+                 'Teacher'
+               else
+                 'Student'
+               end
+        puts "[#{type}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+        menu
+      end
+    end
+  end
```

Comment:
- [ ] KIndly make sure your console app is able to list all created people.

https://user-images.githubusercontent.com/65068771/168397124-d14472a4-a9f4-4cd3-a438-1fec3477a73d.mp4

- [ ] Also sometimes the person is not getting its correct name kindly check in the screencast :point_up: carefully and address the issue.